integration_tests/contracts/contracts/TestICA.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.6.6;
+
+contract TestICA {
+    address constant icaContract = 0x0000000000000000000000000000000000000066;
+
+    function encodeRegister(string memory connectionID) internal view returns (bytes memory) {
+        return abi.encodeWithSignature(
+            "registerAccount(string,string)",
+            connectionID, msg.sender, ""
+        );
+    }
+
+    function callRegister(string memory connectionID) public returns (bool) {
+        (bool result, ) = icaContract.call(encodeRegister(connectionID));
+        require(result, "call failed");
+        return true;
+    }
+
+    function delegateRegister(string memory connectionID) public returns (bool) {
+        (bool result,) = icaContract.delegatecall(encodeRegister(connectionID));
+        require(result, "call failed");
+        return true;
+    }
+
+    function staticRegister(string memory connectionID) public returns (bool) {
+        (bool result,) = icaContract.staticcall(encodeRegister(connectionID));
```

Comment:
I am a bit unsure if this is a staiccall or call. The former is not expected to change the blockchain state. But is Cosmos changes a special case?